### ISSUE TITLE
5.0-alpha1 - double clic on a file doesn't have any effect 

### ISSUE THREAD

**Issue by brunetton** [IvanVolosyuk/diskusage#89]
I'm not sure if I remember well but I'm quite sure that in the last version I was using (a lot!) I was able to double clic on an item to open it with default associated app. I miss this feature that was really helpful.

Of course this is subjective but I'd love to see an option to activate it if it was removed on purpose.

Thank you!

**Comment by IvanVolosyuk** [comment]
There is a show button for preview. I never had a double click for opening
as it would be conflicting with navigation: expand / collapse.

**Comment by brunetton** [comment]
I disagree with you here. All files explorer in the world use the same action to open a directory and a file. It's natural to have this kind of behavior and it's also used in (all?) disk usage apps, for example: https://play.google.com/store/apps/details?id=com.rjhartsoftware.storageanalyzer

**Comment by IvanVolosyuk** [comment]
I understand you. I just stated that I have never had this
functionality in DiskUsage.
